### PR TITLE
packet support

### PR DESCRIPTION
**Pull request by will-chan**

### CODE REVIEW

File: code/implementation/docker/machine/src/main/resources/schema/base/packetConfig.json.d/packet.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "resourceFields": {
+        "apiKey": {
+            "type": "string",
+            "nullable": false,
+	        "required": true
```
Comment (cjellick) [inline code/implementation/docker/machine/src/main/resources/schema/base/packetConfig.json.d/packet.json]: Tab characters are throwing whitespace off

File: code/implementation/docker/machine/src/main/resources/schema/base/packetConfig.json.d/packet.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        },
+        "projectId": {
+            "type": "string",
+            "nullable": false,
+	        "required": true
```
Comment (cjellick) [inline code/implementation/docker/machine/src/main/resources/schema/base/packetConfig.json.d/packet.json]: Want to add minLength for all these fields?

**Comment by cjellick** [comment]
@will-chan For the packet fork of docker-machine, we should maintain that.
We need to:
1. Obtain packet's changes as a single patch
2. clone docker-machine to rancherio
3. Fork off of the latest tagged release (0.2.0 at this point, right?)
4. Apply packet's patch.
5. Build, test, and release our own binary.

File: code/implementation/docker/machine/src/main/resources/schema/base/packetConfig.json.d/packet.json (hunk 2 above)
Comment (will-chan) [inline code/implementation/docker/machine/src/main/resources/schema/base/packetConfig.json.d/packet.json]: No need.  Packet can establish their own validation for projectId and apiKey.

**Comment by cjellick** [comment]
LGTM

**Comment by cjellick** [comment]
Merged with 1 lgtm after discussing with will since this is a simple non-code change

**Pull request merged**